lib/core/src/Cardano/Wallet/Api/Server.hs

```diff
@@ -1499,6 +1512,21 @@ data ErrNoSuchEpoch = ErrNoSuchEpoch
     , errCurrentEpoch :: W.EpochNo
     } deriving (Eq, Show)
 
+getNetworkClock :: NtpClient -> Handler ApiNetworkClock
+getNetworkClock ntpClient = do
+    ntpS <- liftIO $ ntpQueryBlocking ntpClient
+    case ntpS of
+        NtpSyncPending ->
+            pure $ ApiNetworkClock $
+            ApiNtpStatus NtpSyncingStatusPending Nothing
+        NtpSyncUnavailable ->
+            pure $ ApiNetworkClock $
+            ApiNtpStatus NtpSyncingStatusUnavailable Nothing
+        (NtpDrift (NtpOffset (Microsecond ms))) ->
+            pure $ ApiNetworkClock $
+            ApiNtpStatus NtpSyncingStatusAvailable
+            (Just $ Quantity $ fromIntegral ms)
```

Comment:
(optional minor code arrangement issue) I feel like all of this function, except for the `liftIO`, and all of the types could go into the separate NTP module as a helper function `getNtpStatus :: NtpClient -> IO NtpStatus`.

Reply:
good idea. I even went further. Localized all ouroboros's ntp-client imports in Nework.Ntp, use re-exports, and hence remove ntp-client dependancies from ntp-client